composer.json

```diff
@@ -8,7 +8,7 @@
         "dpc-sdp/tide_media": "^3.0.0",
         "drupal/select2": "^1.7",
         "oomphinc/composer-installers-extender": "^2.0",
-        "select2/select2": "^4.0"
+        "npm-asset/select2": "^4.0"
```

Comment:
This is still available on packagist under your existing `select2/select2` namespace so why is this change here?

Reply:
This one was just to match the same package used in the projects root composer e.g content-refe, content-vic. Also it matches the installation instruction given in the contrib module as well - https://www.drupal.org/project/select2
"After this you can install the library with "composer require oomphinc/composer-installers-extender npm-asset/select2" and the library will be downloaded into the libraries folder."

So thought better to consistent and make it same as the projects root composer.

Reply:
Yeah no worries, I'm sure they're the same anyway - all good